galite-core/src/main/kotlin/org/kopi/galite/visual/dsl/form/FormField.kt

```diff
@@ -371,7 +444,11 @@ open class FormField<T>(internal val block: Block,
    * Returns true if the field is never displayed
    */
   val isInternal: Boolean
-    get() = access[0] == VConstants.ACS_HIDDEN && access[1] == VConstants.ACS_HIDDEN && access[2] == VConstants.ACS_HIDDEN
+    get() = vField.isInternal()
+
+  private val _isInternal = access[0] == VConstants.ACS_HIDDEN
+          && access[1] == VConstants.ACS_HIDDEN
+          && access[2] == VConstants.ACS_HIDDEN
```

Comment:
for more readability i propose to change :
0 to VConstants.MOD_QUERY.MOD_QUERY
1 to VConstants.MOD_QUERY.MOD_INSERT
2 to VConstants.MOD_QUERY.MOD_UPDATE